src/scripts/accesskit.css

```
@@ -50,6 +50,11 @@ body.accesskit-no-user-colours article span[style^="color"] {
   color: inherit !important;
 }
 
+body.accesskit-normal-width-scrollbar,
+html.accesskit-normal-width-scrollbar,
+body.accesskit-normal-width-scrollbar div {
```

I think this should make the generated style element unnecessary?
Specificity of the second rule should be 0, 0, 1 if I've counted right

```suggestion
.accesskit-normal-width-scrollbar:is(html, body),
:where(.accesskit-normal-width-scrollbar) div {
```

---

updated the suggestion and comment to make sure we actually have enough specificity to not rely on CSS insertion order 😅